### PR TITLE
Rtinianov smoke

### PR DESCRIPTION
**Pull request by nolag**

### CODE REVIEW

File: core/services/relay/evm/decoder.go [hunk 1]
```diff
@@ -53,7 +55,8 @@ func (m *decoder) GetMaxDecodingSize(ctx context.Context, n int, itemType string
 func extractDecoding(info *codecEntry, raw []byte) (any, error) {
 	unpacked := map[string]any{}
 	if err := info.Args.UnpackIntoMap(unpacked, raw); err != nil {
-		return nil, commontypes.ErrInvalidEncoding
+		argsJson, _ := json.Marshal(info.Args)
```
Comment (jmank88) [inline core/services/relay/evm/decoder.go:58]: Is this secondary error OK to ignore?
Did the linter notice this?

Comment (nolag) [reply core/services/relay/evm/decoder.go:58]: linter didn't seem to care.  I think it's safe.  The type is serializable and if that was to somehow change,  worst-case the string is empty.

**Comment by ilija42** [comment]
files diff got wrecked with the merge

File: core/services/relay/evm/decoder.go (hunk 1 above)
Comment (jmank88) [reply core/services/relay/evm/decoder.go:58]: But why is it ok for that information to be lost in that case? Can you leave a comment?
Or, could it make sense to swap in something like `%#v` instead? Why is JSON even involved?

Comment (nolag) [reply core/services/relay/evm/decoder.go:58]: ok, I can do %#v.  Wasn't sure how friendly it would be by comparison, but it should be good enough.

**Pull request closed without merging**